# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/cosmic-osd/hotfix-api.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- cosmic-osd-epoch-1.0.0-beta.2/src/components/app.rs.vanilla	2025-10-20 20:21:15.558892239 +0200
+++ cosmic-osd-epoch-1.0.0-beta.2/src/components/app.rs	2025-10-20 20:27:05.501517073 +0200
@@ -745,7 +745,7 @@
             event::Event::Window(iced::window::Event::Resized(s)) => Some(Msg::Size(s)),
             event::Event::PlatformSpecific(event::PlatformSpecific::Wayland(wayland_event)) => {
                 match wayland_event {
-                    event::wayland::Event::OverlapNotify(event) => Some(Msg::Overlap(event)),
+                    event::wayland::Event::OverlapNotify(event, ..) => Some(Msg::Overlap(event)),
                     wayland::Event::Layer(LayerEvent::Unfocused, ..) => Some(Msg::Cancel),
                     _ => None,
                 }
--- cosmic-osd-epoch-1.0.0-beta.2/src/components/osd_indicator.rs.vanilla	2025-10-20 20:30:56.077937897 +0200
+++ cosmic-osd-epoch-1.0.0-beta.2/src/components/osd_indicator.rs	2025-10-20 20:31:02.159426441 +0200
@@ -123,7 +123,6 @@
         let mut cmds = vec![];
         cmds.push(get_layer_surface(SctkLayerSurfaceSettings {
             id,
-            pointer_interactivity: false,
             keyboard_interactivity: KeyboardInteractivity::None,
             namespace: "osd".into(),
             layer: Layer::Overlay,
